java/ql/src/semmle/code/java/frameworks/play/PlayHTTPRequestHeader.qll

```diff
@@ -0,0 +1,41 @@
+import java
+
+/**
+<<<<<<< HEAD
```

Comment:
This looks like a merge conflict resolution gone wrong. Could you update this file and remove `<<<HEAD` , etc.

Reply:
@aibaars apologies for that, it's a blind git add done gone wrong.